### ISSUE TITLE
Too many echo_level

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/Kratos#3357]
Hi you all,

I'm having a problem when trying to write some ProblemParameters.json files, related with que quantity of **echo_level** entries that, for some reason, **MUST** be placed. 

For example: 

- In a simple 2D CFD case, echo_level appears inside `problem_data `and in `solver_settings`. 
- In a 2D FSI case, whose ProjectParameters contains informaton for both Fluid and Structural, echo_level appears 5 times:
    - problem_data > echo_level
    - solver_settings > echo_level
    - solver_settings > structure_solver_settings  > echo_level
    - solver_settings > fluid_solver_settings> echo_level
    - solver_settings > mesh_solver_settings> echo_level
- In the new Conjugate heat transfer application, that contains Buoyancy (Fluid + Convection diffusion) and Convection diffusion (yes, again), we have 3 application, so guess what:
    - problem_data > echo_level
    - solver_settings > echo_level
    - solver_settings > fluid_domain_solver_settings> echo_level
    - solver_settings > fluid_domain_solver_settings > fluid_solver_settings> echo_level
    - solver_settings > fluid_domain_solver_settings > thermal_solver_settings> echo_level
    - solver_settings > solid_domain_solver_settings> thermal_solver_settings> echo_level
    - solver_settings > coupling_settings > echo_level

And I still get this error message:

 ```
 File "E:\KratosDeploy\applications\convection_diffusion_application\python_scripts\conjugate_heat_transfer_solver.py", line 28, in __init__
    super(ConjugateHeatTransferSolver, self).__init__(model, custom_settings)
  File "E:\KratosDeploy\kratos\python_scripts\python_solver.py", line 34, in __init__
    self.echo_level = self.settings["echo_level"].GetInt()
RuntimeError: Error: --------- ERROR : --------- getting a value that does not exist. entry string : echo_level

in kratos/includes/kratos_parameters.h:251:Parameters::GetValue
```

Of course, the user interface SHOULD NOT ask the user all this echo_levels. I think that if a user needs to set different values for this variable in more than 5 places, it can be done by editing manually the json file. 

BUT (as a conclusion), would it be possible to have a default value for all the echo_level in kratos? I mean, If I erase (or forget, or consider not necessary) that entry in a json file, **it should never throw an error, just take 0 as default**.

@KratosMultiphysics/kratos-core-team

**Comment by RiccardoRossi** [comment]
@jginternational i specifically want to have  a different level of output for different features...

**Comment by philbucher** [comment]
I just want to add that having a default of zero is easy to do
but I agree with @RiccardoRossi, different solvers need different output-levels

**Comment by RiccardoRossi** [comment]
+1 for defaulting everything to 0

**Comment by loumalouomega** [comment]
And to use the Logger always, to have a consistent output settings

**Comment by rubenzorrilla** [comment]
+1 to default 0

**Comment by philbucher** [comment]
on it

**Comment by jginternational** [comment]
Hooray!

**Comment by jginternational** [comment]
So, This issue is waiting for #3372 to be merged, who is waiting for #3889 wich is merged

**Comment by philbucher** [comment]
I am playing with this atm in another branch.
Once I have succeeded (it looks like I will atm but I cannot promise yet) then I will do #3372 and also other places properly since this is a global problem in Kratos. Sorry for the wait but I would like to solve this properly and for good
